README.md

```diff
@@ -1,17 +1,61 @@
-### License
+SonarQube [![Build Status](https://travis-ci.org/SonarSource/sonarqube.svg?branch=master)](https://travis-ci.org/SonarSource/sonarqube) [![Quality Gate](https://next.sonarqube.com/sonarqube/api/badges/gate?key=org.sonarsource.sonarqube%3Asonarqube)](https://next.sonarqube.com/sonarqube/dashboard?id=org.sonarsource.sonarqube%3Asonarqube) [![Coverage](https://next.sonarqube.com/sonarqube/api/badges/measure?key=org.sonarsource.sonarqube%3Asonarqube&metric=coverage)](https://next.sonarqube.com/sonarqube/component_measures/domain/Coverage?id=org.sonarsource.sonarqube%3Asonarqube)
+=========
 
-Copyright 2008-2017 SonarSource.
+Continuous Inspection
+---------------------
+SonarQube provides the capability to not only show health of an application but also to highlight issues newly introduced. With a Quality Gate in place, you can fix the leak and therefore improve code quality systematically.
```

Comment:
What about "Quality Gate" and "fix the leak" being links to the related blog posts?

Reply:
Good idea. Added to the leak, not found for the gate.

Reply:
@ganncamp is there a blogpost about quality gates we are missing? Or some documentation page explaining the concept?

Reply:
It's got some age on it @sns-seb, but here it is: https://blog.sonarsource.com/quality-gates-shall-your-projects-pass